src/ServiceControlInstaller.Engine/Validation/ConnectionStringValidator.cs

```diff
@@ -14,7 +14,7 @@ class ConnectionStringValidator
         public static void Validate(IServiceControlInstance instance)
         {
             var validator = new ConnectionStringValidator(instance.ConnectionString, instance.ServiceAccount);
-            if (instance.TransportPackage == "SQLServer")
+            if (instance.TransportPackage.ZipName == "SQLServer")
```

Comment:
Why compare the ZipName here and not the transport name? And why not use ignore case comparison like in the rest of the code?

Reply:
Same response as above, ignore case simply because it wasn't there originally. I'll add